pkg/apis/dashboard/migration/migrate_test.go

```diff
@@ -52,6 +52,11 @@ func TestMigrate(t *testing.T) {
 		})
 
 		for targetVersion := range schemaversion.Migrations {
+			// Skip backward migrations
```

Comment:
i opened a new PR to address issues with testing multiple versions: https://github.com/grafana/grafana/pull/100016

Reply:
Thank you so much! 💯